pkgs/development/python-modules/xtensor-python/default.nix

```diff
@@ -10,26 +10,36 @@
 
 buildPythonPackage rec {
   pname = "xtensor-python";
-  version = "0.25.1";
+  version = "0.26.1";
+
+  format = "other"; # Just CMake
 
   src = fetchFromGitHub {
     owner = "xtensor-stack";
     repo = pname;
     rev = version;
-    sha256 = "17la76hn4r1jv67dzz8x2pzl608r0mnvz854407mchlzj6rhsxza";
+    sha256 = "sha256-kLFt5Ah5/ZO6wfTZQviVXeIAVok+/F/XCwpgPSagOMo=";
   };
 
   nativeBuildInputs = [ cmake pybind11 ];
+  checkInputs = [ gtest ]; # find_package fails if we use nativeCheckInputs
```

Comment:
Likely the cmake file is wrong

Reply:
I also find this behaviour weird. They [`find_package(GTest REQUIRED)`](https://github.com/xtensor-stack/xtensor-python/blob/815ffe50838924029bb98c69a480c511d1c14fc8/test/CMakeLists.txt#L79), so it looks like the tests are meant to be run on the Host platform. Should I just drop them from the checkPhase then?

Reply:
Is there any chance that there are some search path-building hooks that account for `checkInputs` but not for `nativeCheckInputs`?

Reply:
> Should I just drop them from the checkPhase then?

I think we rather won't tests that don't work cross than no tests.



> Is there any chance that there are some search path-building hooks that account for `checkInputs` but not for `nativeCheckInputs`?

Could certainly be the case but I don't have any concreate ideas.

Reply:
> I think we rather want tests that don't work cross than no tests.

Fair. Users can always `overridePythonAttrs` and disable tests if they want to cross

Reply:
This behavior is because `checkInputs` are added to `buildInputs` and `nativeCheckInputs` are added to `nativeBuildInputs`.
Since `gtest` is linked in the buildPhase, it needs to be included in buildInputs.

Reply:
Same here.